src/packages/get-platform/src/getPlatform.ts

```diff
@@ -176,6 +193,9 @@ export async function getPlatform(): Promise<Platform> {
   if (platform === 'netbsd') {
     return 'netbsd'
   }
+  if (platform === 'linux' && arch === 'arm64') {
```

Comment:
We will support only arm64 or arm too (32 I guess?)

Reply:
Currently only `arm64`